internal/types/host/both-host.go

```diff
@@ -0,0 +1,17 @@
+package host
+
+import "github.com/cafebazaar/keepalived-exporter/internal/types"
+
+// BothHost implements KeepalivedCollector for when Keepalived and Keepalived Exporter are both on a same host
+type BothHost struct {
```

Comment:
I think `Keepalived{Host/Container}Collector{Host/Container}` would be a better naming convention.